MH-FiorellaTheme5/source/javascripts/site.js

```diff
@@ -0,0 +1,25 @@
+$('.slider').slick({
+    autoplay: true,
+    dots: true,
+    prevArrow: false,
+    nextArrow: false,
+    responsive: [{
+        settings: {
+            dots: false,
+            arrows: false,
+            infinite: false,
+        }
+    }]
+});
+
+document.getElementById("CloseMenu").addEventListener("click", () => {
+  document.getElementById("logo").className = 'whiteLogo';
+  document.getElementById('mySidenav').classList.remove('openSideNav');
+  document.getElementById('mySidenav').classList.add('closeSideNav');
+});
+
+document.getElementById("OpenMenu").addEventListener("click", () => {
```

Comment:
Mixed double and single quotes.<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).